docs/getting-started/nginx/index.md

```diff
@@ -0,0 +1,150 @@
+# Getting Started - NGINX Ingress
+
+This guide covers how Argo Rollouts integrates with the
+[NGINX Ingress Controller](https://github.com/kubernetes/ingress-nginx) for traffic shaping. 
+This guide builds upon the concepts of the [basic getting started guide](../../getting-started.md).
+
+## Requirements
+- Kubernetes cluster with NGINX ingress controller installed
+
+!!! tip
+    See the [environment setup guide](../setup/index.md) on how to setup a local minikube environment with nginx
+
+## 1. Deploy the Rollout, Services, and Ingress
+
+When NGINX Ingress is used as the traffic router, the Rollout canary strategy must define
+the following mandatory fields:
+
+```yaml
+apiVersion: argoproj.io/v1alpha1
+kind: Rollout
+metadata:
+  name: rollouts-demo
+spec:
+  strategy:
+    canary:
+      # Reference to a Service which the controller will update to point to the canary ReplicaSet
+      canaryService: rollouts-demo-canary
+      # Reference to a Service which the controller will update to point to the stable ReplicaSet
+      stableService: rollouts-demo-stable
```

Comment:
I'm wondering if users will understand that the controller is updating the selector to point at pods of a specific replicaset at this point. If they go from the index to the getting started to this, there is no explanation of how this works.

Reply:
I hope the comment is good enough for them, but I think this is something that should be explained in greater detail in the canary section